### ISSUE TITLE
Using HTML in Post/Page Markdown

### ISSUE THREAD

**Issue by robotsquidward** [zerostaticthemes/hugo-winston-theme#17]
In pages & posts in this theme, inline HTML isn't rendered on the page. This [should be supported by Markdown](https://daringfireball.net/projects/markdown/syntax#html), so I'm wondering if/how this theme is suppressing inline Markdown? Is there any way to disable this behavior?

**Comment by JugglerX** [comment]
You need to configure the markdown parser to render raw html.  See this tutorial https://www.youtube.com/watch?v=igIXqfSxm4k

or add this to the config.toml

```
[markup]
  [markup.goldmark]
    [markup.goldmark.renderer]
      unsafe = true
```

I'll add this to the config in the next release

**Comment by JugglerX** [comment]
Fixed in 9edacfbc30f318fa8fc6e2a347fc9955bd0eb9be
